src/editorialPalette.ts

```diff
@@ -0,0 +1,65 @@
+// ----- Imports ----- //
+
+import { Format, Pillar } from '@guardian/types/Format';
+import {
+    news,
+    opinion,
+    sport,
+    culture,
+    lifestyle,
+} from '@guardian/src-foundations/palette';
+
+
+// ----- Types ----- //
+
+type Colour = string;
+
+
+// ----- Functions ----- //
+
+const svgPrimary = (format: Format): Colour => {
```

Comment:
I would go with something more semantic here rather than tying the name to the current implementation. Maybe`iconFillPrimary`?

Reply:
Yup, good call!